ui/page.js

```diff
@@ -2,6 +2,7 @@ const { merge } = require('lodash');
 const path = require('path');
 const findRoot = require('find-root');
 const { Router } = require('express');
+const notifications = require('../lib/middleware/notifications');
```

Comment:
This is on the page router and not global because it needs the content to have been set, right?

Reply:
maybe, at least that was the intention, but it actually might not matter

Reply:
as content is set in the global router, but it is extended here. But I guess at the point of calling the req.notification function the content will have been merged

Reply:
Yeah, that's fine. You _could_ work around it with some weird `onRequestEnd` stuff, but this is fine.